Packs/RecordedFuture/Integrations/RecordedFuture/RecordedFuture.py

```diff
@@ -410,1199 +138,358 @@ def create_indicator(
-                "rule": a.get("rule", {}).get("name"),
-                "type": a.get("type"),
-                "entities": a.get("entities", []),
-            }
-            for a in resp.get("data", {}).get("results", [])
-        ]
-        if not alerts_context:
-            return {
-                "Type": entryTypes["note"],
-                "Contents": {},
-                "ContentsFormat": formats["json"],
-                "ReadableContentsFormat": formats["markdown"],
-                "HumanReadable": "No results found",
-                "EntryContext": {},
-            }
-        alerts_table = [
-            {
-                "Alert ID": ma["id"],
-                "Alert Title": ma["name"],
-                "Rule": ma["rule"],
-                "Status": ma["status"],
-                "Triggered": ma["triggered"],
-                "Assignee": ma["assignee"],
-            }
-            for ma in alerts_context
-        ]
-        return {
-            "Type": entryTypes["note"],
-            "Contents": resp,
-            "ContentsFormat": formats["json"],
-            "ReadableContentsFormat": formats["markdown"],
-            "HumanReadable": tableToMarkdown(
-                "Recorded Future Alerts",
-                alerts_table,
-                headers=headers,
-                removeNull=True,
-            ),
-            "EntryContext": {
-                "RecordedFuture.Alert(val.ID === obj.id)": createContext(
-                    alerts_context
-                )
-            },
-        }
+            update_alert_status = response.pop('alerts_update_data', None)
+            if update_alert_status:
+                self.client.alert_set_status(update_alert_status)
 
-    def get_links_command(self, entity: str, entity_type: str) -> CommandResults:
-        try:
-            entity_data = self.client.entity_enrich(entity, entity_type, related=False, risky=False)
-            links_data = entity_data.get("data", {}).pop("links")
-            if "error" in links_data:
-                raise Exception(links_data["error"])
-            formated_links = self.__format_links(links_data)
-            scored_links = self.__soar_links(formated_links)
-            markdown = self.__build_links_markdown(entity, scored_links)
-            return self.__build_links_context(scored_links, markdown)
-        except DemistoException as err:
-            if "404" in str(err):
-                return CommandResults(
-                    outputs_prefix="",
-                    outputs={},
-                    raw_response={},
-                    readable_output="No results found.",
-                    outputs_key_field="",
-                )
-            else:
-                raise err
+    def malware_search_command(self) -> List[CommandResults]:
+        """Malware search command."""
+        response = self.client.entity_search()
+        return self._process_result_actions(response=response)
```

Comment:
we do handle it in _call()

```
 except DemistoException as err:
            if '404' in str(err):
                return CommandResults(
                    outputs_prefix='',
                    outputs=dict(),
                    raw_response=dict(),
                    readable_output='No results found.',
                    outputs_key_field='',
                )
            else:
                raise err
```

or on the API side we also handle some "No results" cases to provide more details in human readable response (ex. in case of links command - it can be that entity does not exist, or entity does exist but have no links - and we handle it on API side and return distinguishable response for those cases).